tests/drivers/test_elasticsearch.py

```diff
@@ -256,6 +257,68 @@ def test_combination_of_glob_expressions_should_be_concatenated(self):
         self.assertEqual(result, "foo(bar|baz)[^ab].*")
 
 
+class ElasticSearchTestAccessor(unittest.TestCase):
+
+    def setUp(self):
+        self._accessor = bg_elasticsearch._ElasticSearchAccessor()
+        self._old_connect_method = self._accessor.connect
```

Comment:
This is never used. What was your intent backing this?

Why is it different from `client`?